content/en/docs/home/contribute/write-new-topic.md

```diff
@@ -91,39 +91,46 @@ Here's an example of an entry in /_data/tasks.yaml:
 
 ## Including code from another file
 
-To include a code file in your topic, place the code file in the Kubernetes
-documentation repository, preferably in the same directory as your topic
-file. In your topic file, use the `include` tag:
+If you want to include some code in your topic, you can do one of the following:
 
-<pre>&#123;% include code.html language="&lt;LEXERVALUE&gt;" file="&lt;RELATIVEPATH&gt;" ghlink="/&lt;PATHFROMROOT&gt;" %&#125;</pre>
+- Embed the code in your topic directly using the markdown code block syntax.
+  When the provided code is not generic enough for users to try out, or the code
```

Comment:
I don't understand what point or recommendation this sentence is trying to make. Please reword this.